src/sentry/tagstore/base.py

```diff
@@ -219,6 +220,13 @@ def get_tag_keys(self, project_id, environment_id, status=TagKeyStatus.VISIBLE):
         """
         raise NotImplementedError
 
+    def get_tag_keys_for_projects(self, projects, environments, start,
```

Comment:
I wonder if the base class here could have an implementation that just loops over `for project in projects` and calls `get_tag_keys` on all of them and aggregates the results. Then we technically have backwards compatibility for the older versions of tagstore, even if the implementation will be slow.

On the other hand if we are going to deprecate old tagstore anyway, and given that we can't really aggregate the results properly because we can't just add up `uniqueValues` from different projects, and get anything meaningful... its maybe just not worth it.

Reply:
oh i like that idea except yeah, the uniqueValues seems like the data won't be very correct. i'll leave it as is for now, but if it seems worthwhile to do because open source users or someone would be using it, we could consider omitting the `uniqueValues` from the results since i'm not totally sure if the frontend actually needs that data